backend/src/main/java/com/woowacourse/gongcheck/presentation/GuestAuthController.java

```diff
@@ -0,0 +1,28 @@
+package com.woowacourse.gongcheck.presentation;
+
+import com.woowacourse.gongcheck.application.GuestAuthService;
+import com.woowacourse.gongcheck.application.response.GuestTokenResponse;
+import com.woowacourse.gongcheck.presentation.request.GuestEnterRequest;
+import javax.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GuestAuthController {
+
+    private final GuestAuthService guestAuthService;
+
+    public GuestAuthController(final GuestAuthService guestAuthService) {
+        this.guestAuthService = guestAuthService;
+    }
+
+    @PostMapping("/api/hosts/{hostId}/enter")
```

Comment:
```suggestion
@RestController
@RequestMapping("/api")
public class GuestAuthController {

    private final GuestAuthService guestAuthService;

    public GuestAuthController(final GuestAuthService guestAuthService) {
        this.guestAuthService = guestAuthService;
    }

    @PostMapping("/hosts/{hostId}/enter")
```
`/api`같은 경우 공통적으로 대부분의 api에서 사용하기 때문에 `RequestMapping`에서 먼저 필터링해주어도 될 것 같아요